### ISSUE TITLE
Search API vs. Web Scraping

### ISSUE THREAD

**Issue by olivierobert** [Quentin-34/MyGoogleParser#10]
While #6 implements the [Google Custom Search API](https://developers.google.com/custom-search/v1/introduction/), the stored data does not include AdWords data. I am not up-to-date with that API, so I am 100% sure but this data might not be available at all via that API.

There are several ways to get this data, but one solution that does not rely on third-party solutions is to scrap the Google Search Result Page (SERP). Scraping comes with its own difficulty and workaround which you will need to work around creatively 💪

**Comment by Quentin-34** [comment]
Implemented a first solution for scraping the results. I am not able to get the more accurate results regarding number of links & adwords + complete HTML Code. But I am unable to find the data corresponding to number of results & search time.

**Comment by olivierobert** [comment]
The search results details is usually displayed at the top of the page:

![image](https://user-images.githubusercontent.com/696529/140449347-4f6c8d7a-5698-4faa-ae15-67303ab99bda.png)

**Comment by olivierobert** [comment]
Upon review of the codebase, the parsing of this data is still missing, right?

**Comment by Quentin-34** [comment]
Indeed, for some reasons that I cannot explain, it seems that the response does not contain that specific div anymore.
From what I have seen online, people were searching for a div with 'result-stats' as id but I am not able to find it.
